docs/gc.md

```diff
@@ -35,31 +35,34 @@ To prevent applications running out of memory, objects in the Java heap that are
 
 ## Garbage collection policies
 
-OpenJ9 provides several different GC policies, each one specialized for operation with a specific heap configuration and one or more collectors. These policies are summarized in the followng table. Follow the links to read more about individual policies:
+OpenJ9 provides several GC policies that are designed around different application workloads. Each policy operates on a specific heap configuration and one or more collectors. These policies are summarized in the following table. Follow the links to read more about individual policies:
 
-| GC Policy                                        | Heap Configuration              | Collectors                         |
-|--------------------------------------------------|---------------------------------|------------------------------------|
-| [`gencon     `](#gencon-policy-default) (default)| Generational (nursery/tenure)   | Scavenger, Concurrent Mark, Global |
-| [`balanced   `](#balanced-policy)                | Generational (multiple regions) | Incremental Generational           |
-| [`metronome  `](#metronome-policy)               | Segregated                      | Realtime                           |
-| [`optavgpause`](#optavgpause-policy)             | Flat                            | Concurrent Mark, Global            |
-| [`optthruput `](#optthruput-policy)              | Flat                            | Global                             |
-| [`nogc       `](#nogc-policy)                    | --                              | --                                 |
+| GC policy name (option name)                                           |  Heap configuration             | Collectors                         |
+|------------------------------------------------------------------------|---------------------------------|------------------------------------|
+| [Generational concurrent (`gencon`)](#gencon-policy-default) (default) | Generational (nursery/tenure)   | Scavenger, Concurrent Mark, Global |
+| [Balanced (`balanced`)](#balanced-policy)                              | Generational (multiple regions) | Incremental Generational           |
```

Comment:
* Hybrid mostly Concurrent-Incremental Global Mark Phase, based on Incremental-update (Card-dirtying) write barrier, and followed by global STW sweep.
* Evacuating Partial GC that includes both mandatory Eden space and a set of older regions for incremental de-fragmentation

Reply:
'Generational (multiple regions) ' could be also said: Multi-generational, region based

Reply:
This is probably too fine detail for this table, but PGC is also capable of incremental class unloading. For sure this should be mentioned later in detailed description.